aboutDesktop.css

```diff
@@ -0,0 +1,59 @@
+@media screen and (min-width: 768px) {
+  #first-section > div > p {
+    display: none;
+  }
+
+  #first-section > div > h1 {
+    width: 70%;
+    margin: auto;
+    margin-top: 60px;
+  }
+
+  #first-section > div > #content {
+    width: 70%;
+    margin: auto;
+  }
+
+  #first-section > div > #p1 {
+    margin-top: 50px;
+    display: block;
+  }
+
+  #first-section > div > #mail {
+    display: block;
+  }
+
+  #otherPrograms {
+    padding: 30px 0;
+  }
+
+  #otherPrograms > h3 {
+    padding: 20px;
+  }
+
+  #otherPrograms > div {
```

Comment:
- [ ] It's not recommended to use ids as selectors. Please use classes instead.

Reply:
Hello @arslanbisharat 
Is using classes a required or optional change.
Because some of the elements are single elements not group elements and that is why I opted for an ID instead of a class

Reply:
Hi @Elisha-Tetteyfio, Thank you so much for this question. Yes, using IDs is compulsory according to Microverse Standards.

Reply:
You mean classes right? :) 
But my issue is, even when styling a single element uniquely,.....do I still need classes as classes is for groups of elements instead of single elements?